src/main/java/org/apache/commons/compress/archivers/ar/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+@Version("1.14.0")
```

Comment:
I don't think we want to touch all these files for every release. Isn't there a different way to do this?

Reply:
**Git attributes** seems to be the way in a git-only world. Performing [Keyword Expansion](https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes) to create a `$Version$` keyword or similar.

In the meantime, an Interesting bit from an [SO thread](https://stackoverflow.com/a/11535358):

>  You have to be careful, though, because the .gitattributes file is committed and passed around with the project, but the driver (in this case, dater) isn’t, so it won’t work everywhere. When you design these filters, they should be able to fail gracefully and have the project still work properly.